extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -34,22 +41,86 @@ public function discussions() {
 					$postObjects[] = $post->jsonSerialize();
 				}
 
+				if ($wgLanguageCode === 'en') {
+					//This is temporary to render new discusions card on en wikis (templates in php and mustache have the same name)
+					$this->response->setTemplateEngine( WikiaResponse::TEMPLATE_ENGINE_PHP );
+				}
 				$this->response->setCacheValidity( WikiaResponse::CACHE_VERY_SHORT );
 				$this->response->setData( [
 					'title' => wfMessage( 'recirculation-discussion-title' )->plain(),
 					'linkText' => wfMessage( 'recirculation-discussion-link-text' )->plain(),
 					'discussionsUrl' => $discussionsUrl,
 					'posts' => $postObjects,
 				] );
+
 				return true;
 			}
 		}
 
 		return false;
 	}
 
+	public function footer() {
+		global $wgSitename, $wgCityId, $wgLanguageCode;
+
+		if ($wgLanguageCode !== 'en') {
+			//This is temporary to supress MCF for old discussions on non-en wikis;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)